lib/audit/orphanedIncludes.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const glob = require('tiny-glob');
+const path = require('path');
+
+const logger = require('../logger');
+
+const includes = /{%-?\ include\ (['"])([\w/]*)\1/g;
+const variableInclude = /{%-?\ include\ ([\w]+)/;
+
+// Filter for {app,modules}/ top level directories.
+// This is necessary because tiny-glob doesn't return files from all subdirectories: https://github.com/terkelg/tiny-glob/issues/28
+filterAppModules = files => files.filter(file => file.match(/^(app|modules)\//));
+
+normalizePath = (partialName, includedFrom) => {
+    // App directory.
+    if (includedFrom.match(/^app/)) {
+        return `app/views/partials/${partialName}.liquid`;
+    }
+
+    // Modules.
+    // Get module/[moduleName].
+    const includedFromPath = path.join(...includedFrom.split(path.sep).slice(0, 2));
+
+    // Check if file is available in the private folder.
+    const moduleFilePrivatePath = `/private/views/partials/${partialName}.liquid`;
+    if (fs.existsSync(includedFromPath + moduleFilePrivatePath)) {
+        return includedFromPath + moduleFilePrivatePath;
+    }
+
+    // Otherwise return public folder
+    return `${includedFromPath}/public/views/partials/${partialName}.liquid`;
+};
+
+module.exports = {
+    audit: async () => {
+        let results = {};
+
+        let foundVariableInclude = false;
+
+        // @TODO what about ^_.liquid includes?
+
+        // Find all partials in **/*.liquid.
```

Comment:
@Coornail Instead of writing nice comments extract code to functions with meaningful names.